mtmount/Dockerfile

```diff
@@ -0,0 +1,111 @@
+# This docker file cannot fully automate the EUI installation. You must carry on
+# with the installation of the VIPM package installs to complete the installation. 
+
+FROM centos:centos7
+
+LABEL maintainer Andrew Heyer <aheyer@lsst.org>
+
+ENV USER ${USER:-saluser}
+ENV HOME /home/saluser
+USER root
+# Create user and group
+RUN groupadd lsst && adduser -u 1004 -m -g lsst -s /bin/bash saluser
+
+COPY lsst-ts-nexus.repo /etc/yum.repos.d/lsst-ts-nexus.repo
+COPY wandisco-git.repo /etc/yum.repos.d/wandisco-git.repo
+RUN rpm --import http://opensource.wandisco.com/RPM-GPG-KEY-WANdisco
+RUN yum install -y git wget
+RUN yum install -y OpenSpliceDDS-6.9.0
+
+#Install and update various packages needed for Labview, VPM, and be able to
+#compile C++ tma management 
+#RUN yum update
+RUN yum groupinstall -y "Development Tools"
+RUN yum install -y cmake boost-devel.x86_64
+RUN wget http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+RUN rpm -ivh epel-release-latest-7.noarch.rpm
+RUN yum install -y geany java-1.7.0-openjdk-devel
+RUN yum install -y glibc.i686 libstdc++.so.6 libXinerama.i686 && \
+yum upgrade -y gnome-packagekit-common && \
+yum install -y libglvnd-glx-1.0.1-0.8.git5baa1e5.el7.i686
+RUN yum install -y yum-utils device-mapper-persistent-data lvm2
+RUN yum install -y which python3-devel tcl tcl-devel
+
+# Install tma manager NOTE - software is proprietary - must be in private repo
+# Another note this software has not been setup for git modules, and so it only
+# works if you manually pull it from tekniker gitlap repository.
+COPY lsst ${HOME}/lsst
+RUN chown saluser:lsst -R ${HOME}/lsst/
+
+# Setup SAL
+USER saluser
+WORKDIR ${HOME}
+RUN mkdir repos
+WORKDIR /home/saluser/repos
+RUN  git clone https://github.com/lsst-ts/ts_sal.git && \
+     git clone https://github.com/lsst-ts/ts_xml.git 
+ENV LSST_SDK_INSTALL=/home/saluser/repos/ts_sal
+ENV OSPL_HOME=/opt/OpenSpliceDDS/V6.9.0/HDE/x86_64.linux
+ENV LSST_DDS_DOMAIN=citest
+RUN source /home/saluser/repos/ts_sal/setup.env && \
+    mkdir -p $SAL_WORK_DIR && \
+    mkdir -p $TS_SAL_DIR/lib
```

Comment:
If you need SAL, why don't you start with lsstts/develop-env ? It safes a lot of steps..

Reply:
develop-env includes the LSP which is an unnecessary dependency in this case and increases the size of an already large image.